functions.php

```diff
@@ -354,11 +365,12 @@ function independent_publisher_author_comment_reply_link( $link, $args, $comment
 		}
 
 		// If the user provided more than a first name, use only first name
-		if ( strpos( $author, ' ' ) ) {
+		$independent_publisher_general_options = get_option( 'independent_publisher_general_options' );
```

Comment:
This line can be removed; `$independent_publisher_general_options` is not used here.